docs/guide/testing.md

```diff
@@ -162,7 +162,150 @@ You can generally solve this by calling [`pause()`][textual.pilot.Pilot.pause] w
 You can also supply a `delay` parameter, which will insert a delay prior to waiting for pending messages.
 
 
-## Textual's test
+## Textual's tests
 
 Textual itself has a large battery of tests.
 If you are interested in how we write tests, see the [tests/](https://github.com/Textualize/textual/tree/main/tests) directory in the Textual repository.
+
+## Snapshot testing
```

Comment:
I dislike a heading followed by another heading. Could we remove the heading on L172 ?

Reply:
Agreed - done